src/store/chat/utils.ts

```diff
@@ -28,11 +28,11 @@ export const ERROR_DIALOG_CONTENT = {
   },
 };
 
-export function translateJoinRoomApiError(errorCode: JoinRoomApiErrorCode | string, roomAlias?: string) {
+export function translateJoinRoomApiError(errorCode: JoinRoomApiErrorCode | string, domain?: string) {
   const content = ERROR_DIALOG_CONTENT[errorCode] || ERROR_DIALOG_CONTENT[JoinRoomApiErrorCode.UNKNOWN_ERROR];
 
-  if (content.linkPath && content.linkPath.includes('{roomAlias}')) {
-    content.linkPath = content.linkPath.replace('{roomAlias}', roomAlias);
+  if (errorCode === JoinRoomApiErrorCode.ACCESS_TOKEN_REQUIRED && content.linkPath?.includes('{domain}')) {
```

Comment:
Just as an FYI, you don't really need the `includes` check because replace will just do nothing if it doesn't find the string.